frontend/package.json

```diff
@@ -20,25 +25,331 @@
     "@tanstack/react-query-devtools": "^5.69.0",
     "an-array-of-english-words": "^2.0.0",
     "autoprefixer": "^10.4.20",
+    "available-typed-arrays": "^1.0.7",
```

Comment:
WTF is all that new dependencies?
In any project we should never have that much dependencies

I don't know how you get that, but we definitely need a clean here